apps/imtr/src/types/client.ts

```diff
@@ -0,0 +1,47 @@
+type ConfigSimpleType = boolean | string | number;
+
+type ConfigQuestion = {
+  autofill?: string;
+  collection: boolean;
+  description: string;
+  id: string;
+  longDescription: string;
+  options: string[];
+  prio: number;
+  text: string;
+  type: "string";  // geo / imtr-type from feel
+  uuid?: string;
+}
+
+type ClientQuestionGeo = {
```

Comment:
This is unused throughout the app. This is for the future, right?

Reply:
Good catch ;-)
The client types are actually not used at the moment. These types will be the base for imtr-client soon! ⚡